lib/everypolitician/country.rb

```diff
@@ -35,5 +35,21 @@ def legislature(query)
         query.all? { |k, v| l.__send__(k).to_s.downcase == v.to_s.downcase }
       end
     end
+
+    def upper_house
+      @upper_house ||= get_house('upper house')
```

Comment:
I think this would be better named as a query, rather than a command, and probably with a name that more readily explains what it's actually getting. Perhaps something like `most_recent` would be a little clearer?